packages/terra-tabs/src/TabPane.jsx

```diff
@@ -35,11 +35,16 @@ const propTypes = {
    * Indicates if the pane label should only display the icon. When tab collapses into menu the label text will be used.
    */
   isIconOnly: PropTypes.bool,
+  /**
+   * Indicates if the pane has active styles or not.
```

Comment:
This doesn't explain why a consumer would use the isActive prop. Maybe something like

```suggestion
   * If enabled, this prop will apply the `aria-selected` style to the pane.
```